modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportDDG2P.pm

```diff
@@ -169,22 +204,24 @@ sub parse_input_file {
 
   my %headers;
 
-  my $csv = Text::CSV->new ( { binary => 1 } )  # should set binary attribute.
+  my $csv = Text::CSV->new ( { binary => 1, eol => $/, sep_char => ","} )  # should set binary attribute.
             || die ("Cannot use CSV: ".Text::CSV->error_diag ()."\n");
 
   # Get columns headers
   $csv->column_names ($csv->getline ($fh));
-
+  
   # Read through the file and parse out the desired fields
   while (my $content = $csv->getline_hr ($fh)) {
 
     # get data from the line
-    my $symbol  = $content->{'gene symbol'};
-    my $allelic = $content->{'allelic requirement'};
-    my $mode    = $content->{'mutation consequence'};
-    my $phen    = $content->{'disease name'};
-    my $id      = $content->{'disease mim'};
-    my @accns   = split/\;/,$content->{'phenotypes'};
+    my $symbol  = $content->{"gene symbol"};
+    my $allelic = $content->{"allelic requirement"};
+    my $mode    = $content->{"mutation consequence"};
+    my $phen    = $content->{"disease name"};
+    my $id      = $content->{"disease mim"} if $content->{'disease mim'} =~ /[0-9]/;
+    my @accns   = split/\;/,$content->{"phenotypes"};
+    my @pubmeds   = split/\;/,$content->{"pmids"};
```

Comment:
To add the confidence and the mutation consequence you need to add this code here:
```
my $confidence_category = $content->{"confidence category"};
my $mutation_consequence = $content->{"mutation consequence"};
$mutation_consequence =~ s/;/,/g;
```